test/base/test_objective.py

```diff
@@ -160,3 +165,40 @@ def rel_err(eps_):
         objective.check_grad_multi_eps([theta], multi_eps=multi_eps)
     assert result_multi_eps['rel_err'].squeeze() == \
         min(rel_err(_eps) for _eps in multi_eps)
+
+
+def test_aesara(max_sensi_order, integrated):
```

Comment:
docstring

Reply:
there is a docstring? arguments come from fixtures